package.json

```diff
@@ -18,65 +18,42 @@
     "not dead"
   ],
   "scripts": {
-    "dev": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop",
-    "build": "gatsby build",
-    "serve": "gatsby serve",
-    "clean": "gatsby clean",
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "hps": "local-ssl-proxy --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
+    "https-proxy-start": "local-ssl-proxy --source 3010 --target 3000 --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
     "vault": "netlify-plugin-vault-variables",
     "vault:local": "NODE_ENV=development netlify-plugin-vault-variables",
-    "lint": "eslint \"src/**/*.{ts,tsx,js,jsx}\"",
     "lint:fix": "eslint \"src/**/*.{ts,tsx,js,jsx}\" --quiet --fix",
     "typecheck": "tsc --noEmit",
-    "tsc": "tsc",
-    "https-proxy-start": "local-ssl-proxy --source 3010 --target 8000 --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
-    "hps": "local-ssl-proxy --cert ./dev/localhost.pem --key ./dev/localhost-key.pem"
+    "tsc": "tsc"
   },
   "dependencies": {
     "@heroicons/react": "^2.0.18",
+    "@storyblok/react": "^2.1.8",
```

Comment:
This is the Storyblok React SDK which works for multiple frameworks. The previously Gatsby built uses the `gatsby-source-storyblok` which includes a wrapper around this React SDK and is one step behind in getting updates. They have updated @storyblok/react to work with server components which is what we need for the latest approach.